data_diff/diff_tables.py

```diff
@@ -239,12 +236,31 @@ def _diff_tables(self, table1, table2, level=0, segment_index=None, segment_coun
         for res in self._thread_map(list, diff_iters):
             yield from res
 
+    def _diff_tables(self, table1, table2, level=0, segment_index=None, segment_count=None):
+        logger.info(
+            ". " * level
+            + f"Diffing segment {segment_index}/{segment_count}, key-range: {table1.start_key:x}..{table2.end_key:x}"
+        )
+
+        (count1, checksum1), (count2, checksum2) = self._threaded_call("count_and_checksum", [table1, table2])
+
+        if not (count1 or count2):
```

Comment:
I always forget that 0 is falsey in Python... I would probably prefer `count1 == 0 or count2 == 0`